pkg/handler/collector/deps_dev/deps_dev.go

```diff
@@ -535,6 +541,9 @@ func (d *depsCollector) collectAdditionalMetadata(ctx context.Context, pkgType s
 		logger.Debugf("The version key was not found in the map: %v", versionKey)
 		versionResponse, err = d.client.GetVersion(ctx, versionReq)
 		if err != nil {
+			if err := d.metrics.AddCounter(ctx, GetVersionErrorsCounter, 1, pkgType, *namespace, name); err != nil {
+				logger.Errorf("failed to add counter: %v", err)
+			}
```

Comment:
nit: for readability can we disasmbiguate the two `err` variables?

Reply:
👍